src/linear_algebra/finite_dimensional.lean

```diff
@@ -656,6 +656,28 @@ end linear_equiv
 
 namespace finite_dimensional
 
+/--
+Two finite-dimensional vector spaces are isomorphic if they have the same (finite) dimension.
+-/
+theorem nonempty_linear_equiv_of_findim_eq [finite_dimensional K V] [finite_dimensional K V₂]
+  (cond : findim K V = findim K V₂) : nonempty (V ≃ₗ[K] V₂) :=
```

Comment:
Can we get `dim K V = dim K V₂` from `cond` using `findim_eq_dim`, then apply `nonempty_linear_equiv_of_dim_eq`?

Reply:
No, there are universe issues with this, while `V` and `V_2` can be in different universe levels.

Reply:
The point is that `dim K V = dim K W` only makes sense when `V` and `W` are in the same universe.

Reply:
I added `lift_dim_eq` versions to the `dim` lemmas, and used them to golf the `findim` lemmas.